src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -16,16 +16,19 @@
  */
 public class NavigationTests extends NewTestTemplate {
 
+  private static final String ROOT_PATH = "/";
+  private static final String ROOT_PATH_SECTION = "/main/section/";
+  private static final String ROOT_PATH_CATEGORY = "/main/category/";
```

Comment:
shouldn't it be defined inside of class MercurySubpages (or similar class related with curated main pages)